app/src/main/java/com/pvp/app/service/TaskServiceImpl.kt

```diff
@@ -309,6 +312,12 @@ class TaskServiceImpl @Inject constructor(
             }
     }
 
+    private suspend fun isWeekly(activity: SportActivity): Boolean{
+        return userService.user
+            .firstOrNull()?.weeklyActivities
+            ?.contains(activity) ?: false
```

Comment:
I tend to think of the **Elvis** operator as `orElse` method and here would split it into the new line. I guess it is fine like it is. Up to each preferences, but in the end it would be best if we all handled it the same way.